web/src/views/PublicLanding.tsx

```diff
@@ -39,9 +77,101 @@ export const PublicLanding = () => {
+      <HomePageSectionContainer>
+        <Text as="h2" mb={2}>
+          Register a new project
+        </Text>
+        <Text mb={3}>
+          Use this website if you are a Product Owner for a new cloud-native application and are
+          interested in hosting the app on the OpenShift 4 Platform. You can learn about the BCGov's
+          PaaS/OpenShift 4 Platform Service{' '}
+          <StyledExternalLink
+            rel="noopener noreferrer"
+            href="https://developer.gov.bc.ca/topic/featured/Service-Overview-for-BC-Government-Private-Cloud-as-a-ServiceOpenshift-4-Platform"
+            target="_blank"
+          >
+            here
+          </StyledExternalLink>
+        </Text>
+        <Text as="h3" mb={1}>
+          Before you start
+        </Text>
+        <Text>
+          This website is for teams who've attended an onboarding session with the platform team (if
+          you currently host an application on OpenShift, you’ve done this already.) If you haven’t
+          attended an onboarding session, please contact the Platform Director(
+          <a rel="noopener noreferrer" href="mailto:olena.mitovska@gov.bc.ca" target="_blank">
+            olena.mitovska@gov.bc.ca
+          </a>
+          ) to book an onboarding session.
+        </Text>
+
+        <StyledacknowledgeMessage pb={2} active={showWarningMessage}>
+          <Checkbox
+            name="attendedOnboardingSession"
+            type="checkbox"
+            onChange={() => {
+              SetIsAttendedSession(!isAttendedSession);
+            }}
+          />
+          <Text fontSize={[1, 18]}>I confirm I’ve attended an onboarding session.</Text>
+        </StyledacknowledgeMessage>
+        <StyledButton
+          onClick={() => {
+            if (isAttendedSession) {
+              history.push(ROUTE_PATHS.PROFILE_CREATE);
+              keycloak.login({ idpHint: 'idir' });
+              return;
+            }
+            setShowWarningMessage(true);
+          }}
+        >
+          REGISTER A NEW PROJECT (log in with BC IDIR)
+        </StyledButton>
+        {showWarningMessage && (
+          <Text as="p" mt={0} color="red">
+            Please confirm above checkbox before continue.
+          </Text>
+        )}
+      </HomePageSectionContainer>
+
+      <HomePageSectionContainer>
+        <Text as="h3" m={0}>
+          What you will need
+        </Text>
+        <StyledList>
+          <Text as="li">
+            A BC IDIR (you'll be asked to log in with your IDIR to get to the registry)
+          </Text>
+          <Text as="li">A descriptive project name (no acronyms)</Text>
+          <Text as="li">
+            Contact details and Github IDs for a product owner and up to 2 technical leads
+          </Text>
+          <Text as="li">
+            An idea of which common components you will use (see common components list)
+          </Text>
+        </StyledList>
+      </HomePageSectionContainer>
+    </Box>
```

Comment:
In an effort to not block you from moving on, I have submit an adjusted version of the content as a suggestion below. You had the main meat of the page together which was excellent but there were still a few things that could have been improved upon. You will see that I am using the rebass `Flex` as the base as this takes care of the majority of the responsive needs. Wrapping each area in a Box is strictly for organization and not really a requirement. 

I found there were just some bugs involved with your HomePageSectionContainer, such as if you shrink the window vertically the bottom portion of text started to overlap the middle portion. Additionally, you used the Styled link at one point but not for Olenas email so I adjusted that too. I have done thorough testing on this and everything seems pretty happy. 

I am happy to have a call next week to discuss anything you might have questions about.

```suggestion
    <Flex flexDirection="column">
      <Box mb={3}>
        <Text as="h1" mb={3}>
          Welcome to BC Gov's Platform as a Service(PaaS) Project Registry
        </Text>
      </Box>
      <Box mb={3}>
        <Text as="h2" mb={2}>
          Make changes to an existing project
        </Text>
        <Text mb={2}>
          For existing application's hosted on OpenShift 4 Platform. You can update/change all
          project details and request project resource quota increases and downgrades (including
          CPU/RAM/Storage.)
        </Text>
        <StyledButton onClick={() => keycloak.login({ idpHint: 'idir' })}>Login</StyledButton>
      </Box>
      <Box mb={3}>
        <Text as="h2" mb={2}>
          Register a new project
        </Text>
        <Text mb={2}>
          Use this website if you are a Product Owner for a new cloud-native application and are
          interested in hosting the app on the OpenShift 4 Platform. You can learn about the BCGov's
          PaaS/OpenShift 4 Platform Service{' '}
          <StyledExternalLink
            rel="noopener noreferrer"
            href="https://developer.gov.bc.ca/topic/featured/Service-Overview-for-BC-Government-Private-Cloud-as-a-ServiceOpenshift-4-Platform"
            target="_blank"
          >
            here
          </StyledExternalLink>
        </Text>
      </Box>
      <Box mb={3}>
        <Text as="h3" mb={2}>
          Before you start
        </Text>
        <Text mb={2}>
          This website is for teams who've attended an onboarding session with the platform team (if
          you currently host an application on OpenShift, you’ve done this already.) If you haven’t
          attended an onboarding session, please contact the Platform Director(
            <StyledExternalLink
            rel="noopener noreferrer"
            href="mailto:olena.mitovska@gov.bc.ca"
            target="_blank"
          >
            olena.mitovska@gov.bc.ca
          </StyledExternalLink>
          ) to book an onboarding session.
        </Text>
        <StyledacknowledgeMessage pb={2} active={showWarningMessage}>
          <Checkbox
            name="attendedOnboardingSession"
            type="checkbox"
            onChange={() => {
              SetIsAttendedSession(!isAttendedSession);
            }}
          />
          <Text as="h3" fontSize="16px" my={0} lineHeight="normal" ml={2}>I confirm I’ve attended an onboarding session.</Text>
        </StyledacknowledgeMessage>
        <StyledButton
          onClick={() => {
            if (isAttendedSession) {
              history.push(ROUTE_PATHS.PROFILE_CREATE);
              keycloak.login({ idpHint: 'idir' });
              return;
            }
            setShowWarningMessage(true);
          }}
        >
          REGISTER A NEW PROJECT (log in with BC IDIR)
        </StyledButton>
        {showWarningMessage && (
          <Text as="p" color="red">
            Please confirm above checkbox before continue.
          </Text>
        )}
      </Box>
      <Box mb={3}>
        <Text as="h3">
          What you will need
        </Text>
        <StyledList>
          <Text as="li">
            A BC IDIR (you'll be asked to log in with your IDIR to get to the registry)
          </Text>
          <Text as="li">A descriptive project name (no acronyms)</Text>
          <Text as="li">
            Contact details and Github IDs for a product owner and up to 2 technical leads
          </Text>
          <Text as="li">
            An idea of which common components you will use (see common components list)
          </Text>
        </StyledList>
      </Box>
    </Flex>
```